wg-releases/README.md

```diff
@@ -66,6 +66,12 @@ This is done primarily to ensure that there are no open avenues of compromise fo
 
 Meeting notes may be viewed in [meeting-notes](meeting-notes).
 
-## Feature Backport Requests for Major Versions in Beta
-
-With our 12-week major release cadence, we are _not_ considering feature backport requests after the Releases WG meeting of Week 3 of each beta cycle. Backport requests after Week 3 will only be considered if there is a very good reason. For backport requests please submit the link to the PR and reason for backport to the Releases WG agenda for consideration.
+## Feature Backport Requests
+Backporting non-breaking features to supported versions requires the approval of the Releases Working Group.  Any feature PR that is targeting branch(es) other than master should be labeled with a `pending-vote 🗳` label.  The approval process will work as follows:
+1. By requesting backport of a feature, the submitter of the PR is assuming responsibility for this feature on the requested branches.  If a backport PR causes any breakage, it may be removed if the PR submitter does not fix the breakage.
+2. All issues labeled as `pending-vote 🗳` will be reviewed during the Releases Working Group weekly meeting.  Alternatively, the Releases Working Group may vote for issues asynchronously for requests that need a quicker response.
+3. Backport requests require that at least 3 working group members approve the request.  If at least 3 working group members have approved the request and there are no objections from other members, the request will be immediately approved once 3 members have approved.  If there is not unanimous approval of the feature, a majority of the Working Group will be required to approve the request.  Due to the distributed nature of the Releases WG, a backport request that doesn't have consensus may take longer to be approved or rejected until all available members can weigh in.
```

Comment:
Unanswered question of should a member be able to be one of the three votes if they are the person making the request.

Personally I think that's ok